app/src/main/java/com/example/paetz/yacguide/TourbookActivity.kt

```diff
@@ -54,13 +53,28 @@ class TourbookActivity : AppCompatActivity() {
         eProjects
     }
 
+    override fun onCreateOptionsMenu(menu: Menu): Boolean {
+        // Inflate the menu; this adds items to the action bar if it is present.
+        menuInflater.inflate(R.menu.tourbook, menu)
+        return true
+    }
+
+    override fun onOptionsItemSelected(item: MenuItem): Boolean {
+        // Handle action bar item clicks here. The action bar will
+        // automatically handle clicks on the Home/Up button, so long
+        // as you specify a parent activity in AndroidManifest.xml.
+        when (item.itemId) {
+            R.id.action_export -> export()
```

Comment:
I suggest to have two items here for Export and Import, respectively. Doing so, we can get rid of this annoying in-between-Dialog Import/Export.

Reply:
Created a separate ticket for this